src/EPR.RegulatorService.Frontend.Web/Views/RegistrationSubmissions/QueryRegistrationSubmission.cshtml

```diff
@@ -62,8 +62,8 @@
                     </button>
                 </div>
                 <div class="govuk-grid-column-full">
-                    <a href="@Model.BackToAllSubmissionsUrl" class="govuk-link govuk-link--no-visited-state">
-                        @Localizer["QueryRegistrationSubmission.BackToAllSubmissions"]
+                    <a href="@ViewBag.BackToAllSubmissionsUrl" class="govuk-link govuk-link--no-visited-state">
+                        @Localizer["RejectRegistrationSubmission.BackToAllSubmissions"]
```

Comment:
Is this the correct Text to use?  Being fussy - perhaps a common entry?

Reply:
As the values are used in different views they are in different resource files.  Not sure where this should go to be shared?